masonry/src/passes/accessibility.rs

```diff
@@ -89,10 +89,7 @@ fn build_accessibility_tree(
 // --- MARK: BUILD NODE ---
 fn build_access_node(widget: &mut dyn Widget, ctx: &mut AccessCtx) -> Node {
     let mut node = Node::new(widget.accessibility_role());
-    node.set_bounds(to_accesskit_rect(
-        ctx.widget_state.window_layout_rect(),
-        ctx.scale_factor,
```

Comment:
Can we remove `AccessCtx::scale_factor`?

Reply:
Hmm, I see it's used in the `root_widget` accessibility. I wonder if we should somehow make this a property? of the root widget specifically?

It seems like it's only useful for that one place (analogy: the paint method doesn't know about the scale factor)

Reply:
I didn't like the idea of passing the scale factor down through all the `AccessCtx` instances either. I think I've come up with something cleaner now.